### DISCUSSION TITLE
Upgrade problem to V0.27.0

### DISCUSSION THREAD

**Discussion by sbisbilo** [netbox-community/netbox-docker#377] [Getting Help] [answered]
Hello everyone,

I updated to V0.27.0 my Netbox and I've got an error (I attached a screenshot of it). Can you tell what I had missed? Thanks.

Sylvain
![2020-12-18_13h39_56](https://user-images.githubusercontent.com/20032436/102615840-db680980-4136-11eb-9664-2b011742225b.png)

**Comment by cimnine** [comment, marked as answer]
It seems you have only updated the Docker Image, but not the rest of the repository. Either disable the volume mounts for `startup_scripts` in the docker-compose file or update the files to the latest version.

If you cloned the repository initially as described in https://github.com/netbox-community/netbox-docker/wiki/Getting-Started, then you can just run `git pull --ff`.

And if you changed the origin git remote to something else you can still run `git pull https://github.com/netbox-community/netbox-docker.git release`, which should fetch the changes from our repository and merge it into yours.

**Comment by sbisbilo** [comment]
Thanks you so much cimnine you made my day! I disabled the volume mount for startup_scripts
